code/modules/mob/living/carbon/human/species/station/diona/diona.dm

```diff
@@ -160,7 +161,7 @@
 /datum/species/diona/handle_death(var/mob/living/carbon/human/H, var/gibbed = 0)
 	if (!gibbed)
 		// This proc sleeps. Async it.
-		INVOKE_ASYNC(H, /mob/living/carbon/human/proc/diona_split_into_nymphs, TRUE)
+		INVOKE_ASYNC(H, /mob/living/carbon/human/proc/diona_split_into_nymphs, total_health * 0.20)
```

Comment:
that's uh. Not how this works. The value is expected to be false-y or truth-y. The result of computation that's other than 0 is truthy.

Reply:
Removed this part entirely given a check already handled it.